README.md

```diff
@@ -1,11 +1,51 @@
-# Xliff-tasks
-Set of tasks and targets to generate xliff files.
+# xliff-tasks
+
+A set of MSBuild tasks and targets to automatically generate xliff (.xlf) files for localizable resources, and to build satellite assemblies from those xliff files.
+
+## Build Status
 
-Build Status
-------------
 |Windows x64 |
 |:------:|
 |[![][win-x64-build-badge]][win-x64-build]|
+
+## Installing
+
+If you're using the [Roslyn Repo Toolset][roslyn-repo-toolset] then the `XliffTasks` package is already pulled in, and enabled by default.
+
+Otherwise, you'll need to add the dotnet-core feed on MyGet (`https://dotnet.myget.org/F/dotnet-core/api/v3/index.json`) to your nuget.config file, and then add a `PackageReference` for the XliffTasks package, like so:
```

Comment:
I think this is now the blob feed. @tannergooding

Reply:
Blob feed? Sorry, that comment went over my head.

Reply:
`<add key="BlobFeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />`

It may be pushed there and dotnet-core myget. I've lost track. @tannergooding would know for sure.

Reply:
Looks like publishing got broken recently and we don't think that we should point people at blob feed yet. So ignore this with respect to this PR and we'll sort it out separately.